R/utils.R

```diff
@@ -575,7 +575,19 @@ merge_2 <- function(dat_1, dat_2, ...) {
   dat_merged <- merge(dat_1, dat_2, ...)
 }
 
-get_plot_coords <- function(plot) {
+#' Takes a ggplot-plot as input and extracts the range of the x-axis coordinates.
+#'
+#' @keywords internal
+#' @noRd
+#'
+#' @param plot ggplot-plot from which to extract the range of the x-axis coordinates.
```

Comment:
Please write: [ggplot]. This will create an automatic link to the ggplot-Package documentation, in case the documentation is rendered.